### DISCUSSION TITLE
Rejected from Educational Discount

### DISCUSSION THREAD

**Discussion by timhagel** [community/community#42846] [GitHub Education]
I've applied for the GitHub student benefits twice and they have both been rejected for the reason of

> You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I thought maybe I had a VPN on when applying so I tried again to ensure VPNs were off and it still got denied.

**Comment by wubaosheng** [comment]
I met the same issues
